code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -52,37 +52,48 @@
 	new foldable(get_turf(src))
 	qdel(src)
 
-/obj/item/weapon/storage/box/survival //VOREStation Add for Tether Size boxes
-	max_w_class = ITEMSIZE_NORMAL
-	max_storage_space = 6 //This is just the right number, there's no constant.
-
+/obj/item/weapon/storage/box/survival
+	name = "crew survival kit"
+	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and white stripe indicate this one contains oxygen."
+	icon_state = "survival"
 /obj/item/weapon/storage/box/survival/New()
 	..()
 	new /obj/item/clothing/mask/breath(src)
 	new /obj/item/weapon/tank/emergency/oxygen(src)
-	new /obj/item/clothing/glasses/goggles(src)
-
-/obj/item/weapon/storage/box/synth //VOREStation Add for Tether Size boxes
-	max_w_class = ITEMSIZE_NORMAL
-	max_storage_space = 6 //This is just the right number, there's no constant.
-
-/obj/item/weapon/storage/box/synth/New()
-	..()
-	new /obj/item/clothing/glasses/goggles(src)
-
-//VOREStation Edit
+	new /obj/item/clothing/glasses/goggles(src) //VOREStation Add - Goggles for Virgo 3B atmosphere.
+	new /obj/item/weapon/reagent_containers/hypospray/autoinjector(src)
+	new /obj/item/stack/medical/bruise_pack(src)
+	new /obj/item/device/flashlight/glowstick(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/candy/proteinbar(src)
+
+/obj/item/weapon/storage/box/vox
+	name = "vox survival kit"
+	desc = "A box decorated in warning colors that contains a limited supply of survival tools. The panel and black stripe indicate this one contains nitrogen."

```

Comment:
"The panel and black stripe indicate this one contains nitrogen."
Pretty sure vox breathe phoron on our codebase, unless that was changed recently?